test/pstl_offload/memory/interop_allocs.pass.cpp

```diff
@@ -0,0 +1,80 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#if !__SYCL_PSTL_OFFLOAD__
+#error "PSTL offload compiler mode should be enabled to run this test"
+#endif
+
+#include "allocation_utils.h"
+
+// in this translation unit we have overload to USM
+#include <new> // include just to provide the local allocation overload
```

Comment:
```suggestion
// in this translation unit we redirect memory to USM
#include <new> // Necessary to make memory redirection
```

Reply:
"Overload" was dropped, but wording is different.